public/js/menFashion.js

```diff
@@ -0,0 +1,5 @@
+function btnClick() {
```

Comment:
When clicking on the second item the effects apply to the first item!

Reply:
we solve it with the saved fashion feature :D